src/vivarium_public_health/risks/distributions.py

```diff
@@ -56,6 +60,7 @@ def setup(self, builder):
                                                   parameter_columns=['age', 'year'])
         self.parameters = {k: builder.lookup.build_table(v, key_columns=['sex'], parameter_columns=['age', 'year'])
                            for k, v in self._parameters.items()}
+        self.randomness = builder.randomness.get_stream('ensemble_propensity')
```

Comment:
Need a cached series of propensities, and an on_initialize_simulants that generates the propensity for each person as they enter the sim.

Reply:
Modify setup method to take builder and name where name is the `SimulationDistribution` name.

Reply:
Changes made and an nih_cvd simulation run successfully (using the new code)